frontend/app_flowy/packages/flowy_infra_ui/lib/style_widget/text_field.dart

```diff
@@ -135,11 +141,7 @@ class FlowyTextFieldState extends State<FlowyTextField> {
 
   void notifyDidEndEditing() {
     if (!focusNode.hasFocus) {
-      if (controller.text.isEmpty) {
```

Comment:
If remove these codes here, how to save the text to disk when switching to another textField?

<img width="634" alt="image" src="https://user-images.githubusercontent.com/86001920/208288191-37d4a794-e42f-4b67-97be-6e336d2e5a72.png">

Reply:
Added `submitOnLeave` flag to determine whether to submit when focus is lost.